### PR TITLE
Bookstore: initialize project with components

### PR DESCRIPTION
**Pull request by rachidelaid**
> tasks done:
- created a react app.
- installed `react-redux` and `react-router-dom`.
- used the "feature folder" approach.
- used as reusable components.
- added some styling just for the layout.

### CODE REVIEW

File: src/components/Books.js [hunk 1]
```diff
@@ -0,0 +1,21 @@
+import './Books.css';
+
+import BookCard from './BookCard';
+
+const Books = () => {
```
Comment (TedLivist) [inline src/components/Books.js:6]: - [ ] Your component is missing the form for adding books. Kindly create a new form component for adding books and render that component in this one also. The form component should have fields for adding book details and a placeholder button for adding (it does not have to work yet).

File: src/components/BookCard.js [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+
+const BookCard = ({ title, category }) => (
+  <div className="card">
+    <div className="details">
+      <span className="category">{category}</span>
+      <h3 className="title">{title}</h3>
+      <p className="author">suzane collns</p>
+      <div className="actions">
+        <p>Comments</p>
+        <p>Remove</p>
```
Comment (TedLivist) [inline src/components/BookCard.js]: - [ ] Your BookCard should have a placeholder button for removing books. What you have here is a `p` tag. I would recommend you refactor this to a button.

**Review by TedLivist — CHANGES_REQUESTED** [review]
## STATUS: CHANGES REQUIRED ♻️ ♻️

Hi @rachidelaid,

Good job so far!
✔️ Good documentation
✔️ App is generated by CRA
However, there are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Check the comments under the review._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear. And please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by mmsesay — APPROVED** [review]
Hi @rachidelaid, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**